src/config/config.js

```diff
@@ -18,11 +18,7 @@ const config = {
     if (this._secrets._db_url) {
       return this._secrets._db_url;
     }
-    const hostname = cls.getNamespace('hapi-request').get('host');
-    if (hostname.startsWith('localhost')) {
-      return 'http://localhost:5984';
-    }
-    return 'http://couchdb.riglet:5984';
+    return 'http://localhost:5984';
```

Comment:
i can't remember why we bother w/ this magic...the Dockerfile now always sets TWIG_API_DB_URL

Reply:
I didn't see a reason either when I was refactoring this. Glad it is gone.